### ISSUE TITLE
Is there a way to follow links?

### ISSUE THREAD

**Issue by slarrain** [lfos/calcurse#409]
When I get an invite appointment, it usually comes with either a _Zoom_, _Meet_ or _Teams_ link for the Video Conference. Using CalDav with Google Calendar, this information is correctly showed on the **Note** of the _Appointment_. However, there is no current way -at least that I can see- to easily follow the link and open it with the browser.  
So far I have seen 3 ways of viewing the Note of an Appointment:

1. Press `Enter` while on the Appointment. It opens a window that shows the note. However, there is a "Press any key to continue" and any key closes that window. 
![calcurse1](https://user-images.githubusercontent.com/9893834/150624018-e229433c-8eae-4544-b5db-fb47fea17193.png)
IMO, here would be great to have something like `newsboat`, where you press `SHIFT+u` and it gives a list of links to open and you select which one you want.
![Screenshot from 2022-01-22 01-03-00](https://user-images.githubusercontent.com/9893834/150624055-6d9b8f78-962f-4886-a8b3-d8caa461450d.png)
Or like [tuir](https://github.com/proycon/tuir) does in a similar way
![image](https://user-images.githubusercontent.com/9893834/150624124-390924b9-4478-408b-a3a4-dd8da8694034.png)

2. Press `>` while on the Appointment. This open the default `pager`, where you can't open/follow links. I've been looking into more fancy pagers, like [most](https://www.jedsoft.org/most/index.html) and [slit](https://github.com/tigrawap/slit) but none of them seem to have that functionality.

3. Edit with your editor and manually selecting the url. Seems like horribly cumberstone. 

I guess that you could in theory make a script that extracts the url and opens it with your browser and use the pip `|` command that comes with `calcurse` but it is something I haven't explore.

Am I missing something or this Issue effectively a Feature Request?

**Comment by slarrain** [comment]
So, playing around a bit more, I found that the Pipe command `|` only pipes the description or the name of the appointment, and not the notes (where the url usually is located). So it looks like the pipe is not the answer.
I played around with `less`, but didn't get very far.
For the meantime, it looks like the best option is to edit the note in `vim` with `n` and then find the link(`/http`) and then press `gx` which opens the link on the browser. It is not the most straightforward way, but I don't see anything better at the moment. 
Any ideas?

**Comment by slarrain** [comment]
So, since I'm not getting much traction, this is what I came up with. It's not perfect, but it works:

When on an Appointment, if I press `>` the note of the appointment is opened with the system pager, by default `less`. Within less, I can do 
```sh
! urlview %
```
provided I installed `urlview` previously, of course. and that allows me to select all the urls in the appointment's note. A simple `Enter` opens the default web browser which will open zoom/teams/meet whichever the case.

I would much rather have a Hotkey that invokes `urlview` when there is a screen showing the note (pressing `Enter` instead of `>`), but this solution is good enough.

![image](https://user-images.githubusercontent.com/9893834/155590124-74eef22d-6c95-4b27-9151-7e7d3b1a5108.png)
